src/benchmark_runner.cc

```diff
@@ -140,23 +141,81 @@ void RunInThread(const BenchmarkInstance* b, IterationCount iters,
   manager->NotifyThreadComplete();
 }
 
+std::variant<int, double> ParseBenchMinTime(const std::string& value) {
```

Comment:
https://github.com/google/benchmark/blob/main/CMakeLists.txt#L141 <-- C++11 is the only requirement (except under MSVC).

no std::variant until C++17.